src/easylogging++.h

```diff
@@ -4008,10 +4008,23 @@ class VRegistry : base::NoCopy, public base::threading::ThreadSafe {
     return !base::utils::hasFlag(LoggingFlag::DisableVModules, *m_pFlags);
   }
 
+  /// @brief Set replacement name for thread id
+  void setThreadName(const char* name) {
```

Comment:
Couple of more changes needed otherwise this change is essentially useless

We should use this getThreadId when resolving format specifier in defaultLogDispatcher or some way to print this in log (%thread). Provide static method in el::Helpers to set thread name.